format/swf/Data.hx

```diff
@@ -205,13 +205,17 @@ typedef Shape4Data = {
 
 // used by DefineFont
 typedef ShapeWithoutStyleData = {
+	var fillBits: Int;

```

Comment:
We shouldn't store `fillBits` and `lineBits` -- we can always calculate them directly from the number of `fillStyles` and `lineStyles`. I'm not so concerned if the value exactly matches Flash, as long as it is valid.